daniel-robert/server.js

```diff
@@ -121,7 +123,7 @@ app.delete('/articles/:id', function(request, response) {
 
 app.delete('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here... 3,4,5. The article is interacting with Article.prototype.deleteRecord. This is apart of the Delete method of CRUD.
```

Comment:
Not deleteRecord(), think about what does this delete function do and how is it different from the previous one.

Reply:
Since this route takes no param, we are dealing with the .truncateTable() function